tools/cli/admin.go

```diff
@@ -688,6 +688,15 @@ func newAdminClusterCommands() []cli.Command {
 				newDomainCLI(c, false).FailoverDomains(c)
 			},
 		},
+		{
+			Name:    "rebalance",
+			Aliases: []string{"rb"},
+			Usage: "Rebalance cluster will failover managed failover domains to preferred cluster. " +
+				"You can set the preference in domain data with PreferredCluster.",
```

Comment:
When setting PreferredCluster in domain data, is there any check performed on the value for making sure the value is a valid cluster name?

Reply:
currently no. but the update domain will failed

Reply:
This step is manual. I added a step to verify the data